librocksdb_sys/src/lib.rs

```diff
@@ -806,6 +819,45 @@ extern "C" {
                                         valLen: *mut size_t)
                                         -> *const u8;
     pub fn crocksdb_pinnableslice_destroy(v: *mut DBPinnableSlice);
+
+    pub fn crocksdb_user_collected_properties_add(props: *mut c_void,
+                                                  key: *const uint8_t,
+                                                  key_len: size_t,
+                                                  value: *const uint8_t,
+                                                  value_len: size_t);
+
+    pub fn crocksdb_table_properties_collector_factory_create
+        (context: *mut c_void)
```

Comment:
In fact, `context` is not a pointer to `c_void`. You should either define the actual structure of `crocksdb_table_properties_collector_factory_context_t` or expose correct contructor to contruct it.